std/commerce/types/Product.ts

```diff
@@ -0,0 +1,26 @@
+export interface Image {
```

Comment:
This should be here or at deco?

Reply:
Here. 

This is the standard type definition for commerce.

I thought we had agreed that it'd be in Live, so we can import it in repos via `$live`. Do you propose to use `deco-sites/deco` instead and publish it to `deno.land`?

Reply:
Oh, you're right, sorry.